From 0d04efadb8289105d4ab7632c8b6d70b860a22c6 Mon Sep 17 00:00:00 2001
From: Alex Williamson <alex.williamson@redhat.com>
Date: Tue, 5 Nov 2013 15:37:46 +0100
Subject: [PATCH 16/25] vfio-pci: Add dummy PCI ROM write accessor

RH-Author: Alex Williamson <alex.williamson@redhat.com>
Message-id: <20131105153746.16057.36730.stgit@bling.home>
Patchwork-id: 55425
O-Subject: [RHEL7 qemu-kvm PATCH 4/5] vfio-pci: Add dummy PCI ROM write accessor
Bugzilla: 1026550
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

Bugzilla: 1026550
Upstream commit: 64fa25a0efcadda6e8197e8ea578f6117d01bb4b

Just to be sure we don't jump off any NULL pointer cliffs.

Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
Reported-by: Paolo Bonzini <pbonzini@redhat.com>
---
 hw/misc/vfio.c |    6 ++++++
 1 file changed, 6 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/misc/vfio.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/hw/misc/vfio.c b/hw/misc/vfio.c
index e336021..d327b40 100644
--- a/hw/misc/vfio.c
+++ b/hw/misc/vfio.c
@@ -1127,8 +1127,14 @@ static uint64_t vfio_rom_read(void *opaque, hwaddr addr, unsigned size)
     return val;
 }
 
+static void vfio_rom_write(void *opaque, hwaddr addr,
+                           uint64_t data, unsigned size)
+{
+}
+
 static const MemoryRegionOps vfio_rom_ops = {
     .read = vfio_rom_read,
+    .write = vfio_rom_write,
     .endianness = DEVICE_LITTLE_ENDIAN,
 };
 
-- 
1.7.1

